### ISSUE TITLE
Warning on "unused" ignore

### ISSUE THREAD

**Issue by TomasMikula** [HairyFotr/linter#48]
```scala
val x = 1/0 // linter:ignore DivideByZero,YodaConditions
```

should cause a warning about `YodaConditions` not present on the line.
